umbraco-cloud/set-up/project-settings/availability-performance.md

```diff
@@ -0,0 +1,106 @@
+# Availability & Performance
+
+In the Umbraco Cloud Settings menu you can find a page called _Availability & Performance_.
+
+Umbraco Cloud's "Availability & Performance" feature lets you see get an overview of your cloud projects' past and current health. Leveraging Azure metrics data, the page provides users with valuable insights into the availability and performance of the cloud project enabling them to monitor and address any issues that may impact the user experience.
+
+## Overview
+
+Under Settings in the Umbraco Cloud Portal, you'll find Availability & Performance. This is where you find the page that shows visualization and statistics for four sections.
+
+The page consists of the following parts:
+- Time range and granularity selector
+- Panel view
+- Chart and statistics view
+
+![AP-Overview](../images/AP-Overview.png)
+
+Note: The page initially display metrics for the live environment for all Umbraco Cloud projects. More detailed visualization and tools intended for troubleshooting to be added in the future will be restricted to “Standard” and “Professional” project plans.
+
+### Time range and granularity selector
+
+When you enter the page, you will a default visualization of the failed requests for the last 24 hours with a data point set for every fifth minute. You are able to change the time range to a predefined interval or define a specific start and end time. You can also select the granularity of the data points.
+
+![Time-range-selector](../images/AP-time-range-selector.png)
+
+Initially, you will only be able to set the time granularity to “5 minutes”.
+
+### Panel view
+
+The panel selector consists of four tiles, each representing a specific segment of data. The four segments are failed request, App Performance, CPU Usage, and Memory Usage.
+
+![Pane-view](../images/AP-Panel-Selector.png)
+
+Each tile includes relevant statistics and potentially a warning or an error indicator in case there is something you might want to consider.
+
+An error indicator is shown in the following situations:
+- Failed Requests: when one or more server errors have occurred in the selected time range.
+- CPU Usage: when the maximum CPU time has exceeded 100% of the plan quota in a 5 minute period during the selected time range.
+- Memory Usage: when the maximum private time has exceeded 100% the plan quota in a 5 minute period during the selected time range.
+
+A warning is shown in case the CPU Usage or Memory Usage
```

Comment:
@sajumb I feel like something is missing from this sentence. 🤔

Reply:
Yes, it should be: "A warning indicator is shown in the following situations:"

Can you change it?

Reply:
@sajumb Yes, i have changed it :D